test/apiv2/test-apiv2

```diff
@@ -364,7 +364,8 @@ function start_service() {
     echo $WORKDIR
     # Some tests use shortnames; force registry override to work around
     # docker.io throttling.
-    env CONTAINERS_REGISTRIES_CONF=$TESTS_DIR/../registries.conf $PODMAN_BIN \
+#    env CONTAINERS_REGISTRIES_CONF=$TESTS_DIR/../registries.conf
```

Comment:
Is there a reason this is commented out versus deleted?

Reply:
@rhatdan Added comment about issue.